numpy/lib/utils.py

```diff
@@ -193,7 +193,41 @@ def deprecate(*args, **kwargs):
     else:
         return _Deprecate(*args, **kwargs)
 
-deprecate_with_doc = lambda msg: _Deprecate(message=msg)
+
+def deprecate_with_doc(msg):
+    """
+    Decorator to deprecate a function and change its docstring to
+    mention the deprecation. 
```

Comment:
Can you make this fit on a single line? It's extracted into a summary table, so should fit in <79 char

Reply:
Alright, done.